core/src/main/java/lucee/runtime/functions/system/GetApplicationSettings.java

```diff
@@ -135,6 +135,8 @@ public static Struct call(PageContext pc, boolean suppressFunctions) throws Page
 		sct.setEL("triggerDataMember", Caster.toBoolean(ac.getTriggerComponentDataMember()));
 		sct.setEL("sameformfieldsasarray", Caster.toBoolean(ac.getSameFieldAsArray(Scope.SCOPE_FORM)));
 		sct.setEL("sameurlfieldsasarray", Caster.toBoolean(ac.getSameFieldAsArray(Scope.SCOPE_URL)));
+		sct.setEL("mmergeFormUrlAsStruct", Caster.toBoolean(ac.getMmergeFormUrlAsStruct()));
```

Comment:
why "mmergeFormUrlAsStruct" but not "mergeFormUrlAsStruct"?

Reply:
good catch :)